cypress/integration/Popup.spec.ts

```diff
@@ -151,4 +132,243 @@ describe('Popup', () => {
       });
     });
   });
+
+  context('given the [Testing/React/Popups/Popup, MixedPopupTypes] story is rendered', () => {
```

Comment:
These specs used to live in `PopupStack.spec.ts`, but I deleted that file and move the story it was testing. All the functionality requires Popup hooks, so it makes more sense for the stories and specs to live in `Popup` instead of `PopupStack`